packages/calcite-components/src/components/dropdown-group/dropdown-group.tsx

```diff
@@ -74,6 +76,7 @@ export class DropdownGroup {
   //--------------------------------------------------------------------------
 
   connectedCallback(): void {
+    this.el.setAttribute("id", this.el.getAttribute("id") || `dropdown-group-${guid()}`);
```

Comment:
Can `dropdown-group-${guid()}` be created on the instance instead of in connectedCallback? otherwise, this guid will change if the component is moved in the DOM and connectedCallback is called again.

Reply:
This applies to other components in this PR as well.